.gitignore

```diff
@@ -1,2 +1,7 @@
 .idea/*
-*build*/
\ No newline at end of file
+*build*/
+libs/
```

Comment:
Isnt this a problem with the submodule?

Reply:
Sorry, I don't get the point. Please clearify.

Reply:
The right way to handle this seems to be that you can mark the submodules as 'dirty', that way any changes in the submodule won't be shown in the staging area. But this should actualy not be in the .gitignore so I removed it.
See also https://stackoverflow.com/questions/7912022/do-you-ignore-a-git-submodule-in-your-gitignore-or-commit-it-to-your-repo/7912101